### ISSUE TITLE
Statistics

### ISSUE THREAD

**Issue by picolino0** [team-thyme/soundboard#89]
You could add statistics to see how many times a soundbyte has been played, or by how many unique users etc.
Then you can sort the soundbytes based on plays.

**Comment by Mesoptier** [comment]
This was an idea we had a while ago. The main problem with this is that it requires us to keep additional data with the samples. Until now it was enough to just have a folder with sound files where all necessary information could be extracted from the hierarchy and the files themselves.

So, it's quite a big step to make, and will probably take some discussion. (Or @villermen saying "Yes.")

**Comment by picolino0** [comment]
he told me to make it an issue here xD but I agree, you would need to keep extra data, though this can easily be achieved by using a database. I think it'd be really interesting to see which soundbytes are played most often.

**Comment by villermen** [comment]
Yeah. Figured we were going to need one regardless if we are going to add requests. The database should be optional in any case.
